.github/workflows/e2e_tests.yml

```
@@ -0,0 +1,44 @@
+name: e2e-test
+
+on:
+  push:
+    branches:
+      - CI/test/e2e
+  # pull_request:
+  #   branches:
+  #     - CI/test/e2e
+  workflow_dispatch:
+
+jobs:
+  run-e2e-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Compose
+        uses: docker/setup-compose-action@v1
+
+      - name: setup node
```

```suggestion
      - name: Setup node
```

Just a small suggestion, 
for `steps` name use capital letters at the start of first word